app/cp_nigeria/helpers.py

```diff
@@ -287,6 +380,7 @@ def __init__(self, project):
             ).format(*system_assets)
 
         self.project = project
+        self.report_obj = ImplementationPlanContent.objects.get(simulation=self.project.scenario.simulation)
         self.image_path = dict(
             es_schema="static/assets/gui/" + options.schema_name,
```

Comment:
Here I noticed `options` is used instead of `self.options`
```suggestion
            es_schema="static/assets/gui/" + self.options.schema_name,
```